content/building/yaml.md

```diff
@@ -195,7 +247,7 @@ For example, you can write a build script with Dart like this:
 
 **Note on building Android app bundles**
 
-If your app settings in Codemagic have building Android app bundles enabled, we will automatically include a script for generating a signed `app-universal.apk` during the YAML export. This file is required for publishing to Google Play. If you're creating a YAML file from a scratch, add the script below to receive this file:
+If your app settings in Codemagic have building Android app bundles enabled, we will automatically include a script for generating a signed `app-universal.apk` during the YAML export. If you're creating a YAML file from a scratch, add the script below to receive this file:
 
     # fetch codemagic helper scripts
```

Comment:
This block is obsolete and should be replaced. I'll provide correct version soon.

Reply:
The following lines should be removed
```
# fetch codemagic helper scripts
...
~/codemagic-build-scripts/android/generate-universal-apks \
  ...
  --pattern 'build/**/outputs/**/*.aab'
```

And replaced with
```
# generate signed universal apk with user specified keys
universal-apk generate \
      --ks /tmp/keystore.keystore \
      --ks-pass $CM_KEYSTORE_PASSWORD \
      --ks-key-alias $CM_KEY_ALIAS_USERNAME \
      --key-pass $CM_KEY_ALIAS_PASSWORD \
      --pattern 'build/**/outputs/**/*.aab'
```